web/api/formatters/json_wrapper.c

```diff
@@ -207,8 +210,7 @@ void rrdr_json_wrapper_begin(RRDR *r, BUFFER *wb, uint32_t format, RRDR_OPTIONS
             char *label_key = NULL;
             int keys = 0;
             while (pattern && (label_key = simple_pattern_iterate(&pattern))) {
-                uint32_t key_hash = simple_hash(label_key);
-                struct label *current_label;
+                const char *label_value;
```

Comment:
Hello @ktsaou,

I am testing your PR on different scenarios and I observed that during compilation time I am receiving this warning:

```sh
web/api/formatters/json_wrapper.c: In function 'rrdr_json_wrapper_begin':
web/api/formatters/json_wrapper.c:213:29: warning: unused variable 'label_value' [-Wunused-variable]
  213 |                 const char *label_value;
      |      
```
Do you still intend to use this variable? Or can we remove?

Reply:
thanks! removed it!